hermes-console-vue/src/views/subscription/SubscriptionView.vue

```diff
@@ -95,7 +97,9 @@
             :subscription-metrics="subscriptionMetrics"
           />
           <service-response-metrics />
-          <manage-messages-card />
+          <manage-messages-card
+            v-if="isSubscriptionOwnerOrAdmin(roles?.roles.value)"
```

Comment:
1. is `?` needed in all `isSubscriptionOwnerOrAdmin(roles?.roles.value)`?

Reply:
It is, otherwise some tests for this views are failing because some components cannot be rendered because of type error